tests/integration/cattletest/core/test_allocation.py

```diff
@@ -97,9 +97,13 @@ def test_allocation_failed_on_create(super_client, new_context):
     assert c.transitioning == 'error'
     assert c.transitioningMessage == 'Failed to find a placement'
 
-    assert c.allocationState == 'inactive'
+    assert c.allocationState == 'activating'
     assert c.volumes()[0].state == 'removed'
 
+    c = super_client.wait_success(super_client.reload(c.purge()))
```

Comment:
wait_success() is needed because there's a 'purging' transition state.  reload might not be necessary since I think wait_success() already does a reload anyways.